tests/examples/awful/placement/closest_mouse.lua

```diff
@@ -0,0 +1,53 @@
+
+local awful = {placement = require("awful.placement")} --DOC_HIDE
+
+local c = client.gen_fake {x = 20, y = 20, width= 280, height=200, screen =screen[1]} --DOC_HIDE
+
+mouse.coords {x=100,y=100} --DOC_HIDE
+ -- Move the mouse to the closest corner of the focused client
+awful.placement.closest_corner(mouse, {include_sides=true, parent=c})
+mouse.push_history() --DOC_HIDE
```

Comment:
- What's `push_history`?
- Shouldn't the `@foo@`-part of the previous commit only be added here?
- Wouldn't it be nicer to have "proper" unit tests in `spec/` than to have some code whose result isn't really looked at?